### ISSUE TITLE
Enable Sync & WebRefiner

### ISSUE THREAD

**Issue by kurtextrem** [cha0sify/chaoschrome_beta#1]
#1: There is no sync yet, other builds from other builders have it (even in m46).
#2: Not sure if that is feature implemented by the guys at Code Aurora, or by the individual builders. But it is nice to have (WebRefiner = Adblock)

**Comment by cha0sify** [comment]
yes i know and i will implement WebRefiner and also the sync feature, currently i am modifing my jenkins build system. when its ready (almost done) i will start working on features, i have a big list of things i want to change :D 

I just moved into my new apartment, so i am a bit busy, stay tuned :) 

thanks for using my build and providing feedback :)

**Comment by kurtextrem** [comment]
No worries, great to know that you're working on it.

Reallife goes first. Where did you move to? :D
